Assignment5Objects/src/assignment5objects/MainProgram.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * @author Samantha Shurie
+ * Assignment 5: Working with Objects
+ */
+package assignment5objects;
+
+
+public class MainProgram {
+   
+    /**
+     * Main method implements both the RetailItem and Car classes. This method
+     * creates a new instance of the Car class and increases/decreases it's speed
+     * field. It also creates three instances of the RetailItem class and prints
+     * their fields.
+     * @param args 
+     */
+    public static void main(String[] args) {
+        int currentSpeed = 0;   //used to retrieve speed value of car object
+         
+        println("EXERCISE 2: CAR CLASS\n");
+        
+        Car car = new Car(2010, "Honda");
+
+        println("\tAccelerating...");
+
+        for(int i = 0; i < 5; i++) {
+            car.accelerate();
+            currentSpeed = car.getSpeed();
```

Comment:
While there is nothing wrong with assigning `currentSpeed` to `car.getSpeed()` and then using that variable in its output. We can avoid having this long lived variable in your program by using using the invocation of `car.getSpeed()` by value. So it would look like something like this.

```java
for(int i = 0; i < 5; i++) {
  car.accelerate();
  //currentSpeed = car.getSpeed();
  println("\t\tSpeed is now " + car.getSpeed() + ".");
}
```